### PR TITLE
Stub the application copyright framebuffer api

### LINKED ISSUE #778: am IApplicationFunctions: 100 (InitializeApplicationCopyrightFrameBuffer) not implemented
am IApplicationFunctions: 100 (InitializeApplicationCopyrightFrameBuffer) is not implemented.

## Service description:
[Switchbrew](https://switchbrew.org/wiki/Applet_Manager_services#InitializeApplicationCopyrightFrameBuffer)
```
Takes an input TransferMemory handle, an s32 width, an s32 height, an u64 tmem_size, and no output.

tmem_size must be 0x40000-byte aligned. width must be 1-1280, and height must be 1-720.

User-processes create the tmem with an user-specified buffer with permissions=0. width = 1280 and height = 720.

This is used as an overlay for screenshots.
```

## Required by:
* https://github.com/Ryujinx/Ryujinx-Games-List/issues/181

### PR DESCRIPTION
**Pull request by Thog**
As we currently don't support multi layers on vi/nvnflinger, this PR
implement a stub of this API.

Close #778, #779 and #780

### CODE REVIEW

File: Ryujinx.HLE/HOS/Services/Am/AppletOE/ApplicationProxyService/ApplicationProxy/IApplicationFunctions.cs [hunk 1]
```diff
@@ -168,6 +168,47 @@ public ResultCode SetGamePlayRecordingState(ServiceCtx context)
...
+        // SetApplicationCopyrightImage(buffer<bytes, 0x45> frame_buffer, s32 x, s32 y, s32 width, s32 height, s32 window_origin_mode)
+        public ResultCode SetApplicationCopyrightImage(ServiceCtx context)
+        {
+            long  frameBufferPos   = context.Request.SendBuff[0].Position;
+            long  frameBufferSize  = context.Request.SendBuff[0].Size;
+            int   x                = context.RequestData.ReadInt32();
+            int   y                = context.RequestData.ReadInt32();
+            int   width            = context.RequestData.ReadInt32();
+            int   height           = context.RequestData.ReadInt32();
+            uint  windowOriginMode = context.RequestData.ReadUInt32();
```
Comment (Cyuubi) [inline Ryujinx.HLE/HOS/Services/Am/AppletOE/ApplicationProxyService/ApplicationProxy/IApplicationFunctions.cs]: `uint  w`, 2 spaces?

Comment (Thog) [reply Ryujinx.HLE/HOS/Services/Am/AppletOE/ApplicationProxyService/ApplicationProxy/IApplicationFunctions.cs]: it's aligned to the other variable names ^^

Comment (jduncanator) [reply Ryujinx.HLE/HOS/Services/Am/AppletOE/ApplicationProxyService/ApplicationProxy/IApplicationFunctions.cs]: I think he means that you can have one less space in all the variable names, and they'd all still be aligned

Comment (jduncanator) [reply Ryujinx.HLE/HOS/Services/Am/AppletOE/ApplicationProxyService/ApplicationProxy/IApplicationFunctions.cs]: I think they mean that you can have one less space in all the variable names, and they'd all still be aligned

**Pull request merged**